src/Apps/Legion/Components/WizardShell/Internal/ModifyStep/ModifyStep.stories.tsx

```diff
@@ -23,7 +23,7 @@ type ModifyStepStory = ComponentStory<typeof ModifyStep>;
 const Template: ModifyStepStory = (args) => {
     return (
         <WizardDataContextProvider
-            initialState={GET_DEFAULT_MOCK_WIZARD_DATA_CONTEXT('Dairy')}
+            initialState={GET_DEFAULT_MOCK_WIZARD_DATA_CONTEXT()}
```

Comment:
Is Dairy no longer used or why remove it here?

Reply:
it's the default if not provided.